python/lsst/obs/base/instrument.py

```diff
@@ -141,6 +179,123 @@ def applyConfigOverrides(self, name, config):
+
+        Parameters
+        ----------
+        butler : `lsst.daf.butler.Butler`
+            Butler to receive these calibration datasets.
+        """
+
+        datasetType = DatasetType("camera", ("instrument", "calibration_label"), "Camera",
+                                  universe=butler.registry.dimensions)
+        butler.registry.registerDatasetType(datasetType)
+        unboundedDataId = addUnboundedCalibrationLabel(butler.registry, self.getName())
+        camera = self.getCamera()
+        butler.put(camera, datasetType, unboundedDataId)
+
+    def writeStandardTextCuratedCalibrations(self, butler):
+        """Write the set of standardized curated text calibrations to
+        the repository.
+
+        Expected to be called from an instrument subclass
+        ``writeCuratedCalibrations`` implementation.
+
+        Parameters
+        ----------
+        butler : `lsst.daf.butler.Butler`
+            Butler to receive these calibration datasets.
+        """
+
+        for datasetTypeName in self.standardCuratedDatasetTypes:
+            # We need to define the dataset types.
+            if datasetTypeName not in StandardCuratedCalibrationDatasetTypes:
+                raise ValueError(f"DatasetType {datasetTypeName} not in understood list"
+                                 f" [{'.'.join(StandardCuratedCalibrationDatasetTypes)}]")
+            definition = StandardCuratedCalibrationDatasetTypes[datasetTypeName]
+            datasetType = DatasetType(datasetTypeName,
+                                      universe=butler.registry.dimensions,
+                                      **definition)
+            self._writeSpecificCuratedCalibrationDatasets(butler, datasetType)
+
+    def _writeSpecificCuratedCalibrationDatasets(self, butler, datasetType):
+        """Write standardized curated calibration datasets for this specific
+        dataset type from an obs data package.
+
+        Parameters
+        ----------
+        butler : `lsst.daf.butler.Butler`
+            Gen3 butler in which to put the calibrations.
+        datasetType : `lsst.daf.butler.DatasetType`
+            Dataset type to be put.
+
+        Notes
+        -----
+        This method scans the location defined in the ``obsDataPackageDir``
+        class attribute for curated calibrations corresponding to the
+        supplied dataset type.  The directory name in the data package much
+        match the name of the dataset type. They are assumed to use the
+        standard layout and can be read by
+        `~lsst.pipe.tasks.read_curated_calibs.read_all` and provide standard
+        metadata.
+        """
+        if self.obsDataPackageDir is None:
+            # if there is no data package then there can't be datasets
+            return
+
+        calibPath = os.path.join(self.obsDataPackageDir, self.policyName,
+                                 datasetType.name)
+
+        if not os.path.exists(calibPath):
+            return
```

Comment:
I was wanting to use the data package as the source of truth rather than have to also edit something in the obs package. obs_base defines the set of all supported calibrations in the data package and the specific data package defines which ones are relevant to that instrument.

Reply:
I guess if the tests are thorough enough, typo-ed paths shouldn't be a problem?

Reply:
If you have a typo-ed path in your obs_x_data package you won't need tests to tell you you've got a problem because nothing you do will work if you are relying on those calibrations.